### PR TITLE
Add region dashboard

### PR DESCRIPTION
**Pull request by sorja**

### CODE REVIEW

File: client/pages/Dashboard/ForestOwnership/ForestOwnership.tsx [hunk 1]
```diff
@@ -14,12 +14,13 @@ import { ChartColors, commonOptions } from '../utils/preferences'
...
   const countryIso = useCountryIso()
+  const isIsoCountry = Areas.isISOCountry(countryIso)
+
   const i18n = useTranslation()
   const section = 'forestOwnership'
-  const isIsoCountry = Areas.isISOCountry(countryIso)
   const unit = isIsoCountry ? i18n.t<string>('unit.haThousand') : i18n.t<string>('unit.haMillion')
   const column = '2015'
-  const tableName = 'forestOwnership'
+  const tableName = isIsoCountry ? 'forestOwnership' : 'aggregate'
```
Comment (minotogna) [inline client/pages/Dashboard/ForestOwnership/ForestOwnership.tsx:23]: Too many repeated strings .
If I remember correctly I added an enum TableNames
Can u add those table names (including 'aggregate' ) there ?

File: client/pages/Dashboard/NaturallyRegeneratingForest/NaturallyRegeneratingForest.tsx [hunk 2]
```diff
@@ -36,7 +37,11 @@ const NaturallyRegeneratingForest = () => {
       label: i18n.t<string>(`statisticalFactsheets.rowName.${variable}`),
       unit,
 
-      data: columns.map((column) => tableData?.[countryIso][tableName][column][variable].raw),
+      data: columns.map((column) => {
+        const raw = tableData?.[countryIso][tableName][column][variable].raw
```
Comment (minotogna) [inline client/pages/Dashboard/NaturallyRegeneratingForest/NaturallyRegeneratingForest.tsx:41]: Use TableDatas to get data in all charts if applicable

Comment (sorja) [reply client/pages/Dashboard/NaturallyRegeneratingForest/NaturallyRegeneratingForest.tsx:41]: Not applicable in this case

File: client/pages/Dashboard/ForestOwnership/ForestOwnership.tsx (hunk 1 above)
Comment (minotogna) [reply client/pages/Dashboard/ForestOwnership/ForestOwnership.tsx:23]: value_aggregate sounds good

File: client/pages/Dashboard/NaturallyRegeneratingForest/NaturallyRegeneratingForest.tsx (hunk 2 above)
Comment (minotogna) [reply client/pages/Dashboard/NaturallyRegeneratingForest/NaturallyRegeneratingForest.tsx:41]: Why not ?

Comment (sorja) [reply client/pages/Dashboard/NaturallyRegeneratingForest/NaturallyRegeneratingForest.tsx:41]: TableDatas expects `Row`, `Col`, `Table` etc, we have the actual values for those `variable`, `column`, `tableName`,

**Review by minotogna — APPROVED** [review]

**Pull request merged**